types/account/index.d.ts

```diff
@@ -15,6 +15,20 @@ export interface ZimletProp {
 	_content: string;
 }
 
+export type GetRightsResponse = {
```

Comment:
Not sure about the place of this type. @CMazzilli since you also managed the types lately, where do you think we should place this type to avoid the import cycle?